docs/operations/versioning.md

```diff
@@ -0,0 +1,305 @@
+ 
+ - With `Version Maintenance` enabled: Same as for `Version Management`, but make sure to deprecate the `supported` version of the particular minor version (in this example: 1.18.3).
+ 
+* * *
+***The `supported` version should be deprecated.***
+
+**Example**: Version 1.18.3 should be deprecated.
+
+ - Deprecate version 1.18.3 by setting the classification `deprecated`, optionally with a expiration date.
+ 
+ - `Version Management` enabled: By just adding a new version e.g 1.18.4 without a classification, Version 1.18.3 will be deprecated and 1.18.4 becomes the new `supported` version.  
+   However, is there no higher non-preview version available, version 1.18.3 will be reset to `supported` without an expiration date. Having one `supported` version is enforced by the `Version Management`.
+ 
+ - With `Version Maintenance` enabled:  Similar as for `Version Management`. Having one `supported` version per `maintained` minor version is enforced. 
+ 
+* * *
+**The current `supported` version has problems and should be replaced with the previous `supported` version.**
+
+**Example**: Version 1.18.3 has problems.
+
+a) Version is not in use by any Shoot.
+ - delete the version from the CloudProfile. 
+ - Manually change the classification of the previously `supported` version.
+ - `Version Management` and `Version Maintenance`: nothing else has to be done. 
+
+b) Shoots are already running the erroneous version
+ -  Make sure there is a higher non-preview version available in the CloudProfile.
+ -  Manually change the classification of the erroneous version to `deprecated` with an `expiration date` in the past. Change the version to `supported` again.
+ - `Version Management` and `Version Maintenance`: The current `supported` version needs to be labeled as `preview` with an `expiration date` in the past. 
+    Because one supported version is enforced, the latest `deprecated` version will automatically be classified as `supported` again.
+    Shoots running the `preview` version with an `expiration date` in the past will be forcefully updated. 
+    
+
+c) Shoots are already running the erroneous version and it is the overall latest version:
+ - Shoots cannot upgrade to a newer version as there is none available. Also a downgrade is not supported. 
+   To reduce the blast-radius, label the version as `deprecated` with an `expiration date` in the past to prevent `auto-updates`, defaulting and new cluster creation with that version.
+   The operator has to wait for a newer patch version or delete the Shoot clusters in question. Only then, the version can be deleted from the CloudProfile.
+ - `Version Management` and `Version Maintenance`: same approach, but label the version as `preview`, otherwise the automation will reconcile it to be the `supported` version.
```

Comment:
```suggestion
 - `Version Management` and `Version Maintenance`: same approach, but label the version as `preview`, otherwise the controller will reconcile it to be the `supported` version.
```

Reply:
Will it automatically mark the next-lower patch version as `supported` again? What if it has had problems as well?